community.adoc

```diff
@@ -4,10 +4,27 @@ title: Community
 permalink: /community/
 ---
 
-## Welcome
+# Welcome to the Kaskada community!
+
+The Kaskada project was open-sourced in 2023. As a growing community, we're here to explore new ways to interact with data at scale and have some fun along the way. We build in the open and use the channels below for communication.
 
 ## Discussions
 
-## Events
+For news and announcements, follow us on Twitter: https://twitter.com/kaskadainc[@kaskadainc].
+
+Join the Stargate group on LinkedIn: https://www.linkedin.com/company/kaskadainc[@kaskadainc].
+
+
+## Contributing
+
+Nothing makes us happier than community contributions. See the https://github.com/kaskada-ai/kaskada/blob/main/README.md[README] in the main Kaskada repository to get a grasp of the project layout and reference the https://github.com/kaskada-ai/kaskada/blob/main/CONTRIBUTING.md[CONTRIBUTING] file for detailed steps.
+
+Use the https://github.com/kaskada-ai[GitHub issue tracker] to file bugs and feature requests.
+
+Acceptance of the DataStax contributor license agreement (CLA) is required in order for us to be able to accept your code contribution. You will see this on the standard pull request checklist for the Kaskada repositories.
```

Comment:
AsciiDoc recommends one sentence per line:
https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line

It makes it much easier to deal with changes since the diffs emphasize the sentence that changed not the huge ball of words.